decidim-budgets/spec/system/admin_manages_project_permissions_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Admin manages project permissions", type: :system do
+  let(:manifest_name) { "budgets" }
+  let(:budget) { create(:budget, component: current_component) }
+  let!(:project) { create(:project, budget:) }
+
+  include_context "when managing a component as an admin"
+
+  before do
+    switch_to_host(organization.host)
+    login_as user, scope: :user
+    visit_component_admin
+    within find("tr", text: translated(budget.title)) do
+      page.find(".action-icon--edit-projects").click
+    end
+    within find("tr", text: translated(project.title)) do
+      page.find(".action-icon--permissions").click
+    end
+  end
+
+  it "Saves the permissions" do
```

Comment:
For now I'd leave it as is, but in the future we should reuse the ["manage resource permissions" shared example](https://github.com/decidim/decidim/blob/develop/decidim-initiatives/spec/shared/manage_resource_permissions_examples.rb). I say not do it now because as far as I see it's only used in decidim-initiatives and for using it on other modules we probably need to do a bit of extra work.